cli.py

```diff
@@ -0,0 +1,161 @@
+import datetime
+import fire
+import json
+import os
+import requests
+
+def time_now():
+    datetime_now = datetime.datetime.now(datetime.timezone.utc)
+    return datetime_now.strftime('%a, %d %b %Y %H:%M:%S GMT')
+
+def safe_mkdir(dir):
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+
+def read_json(file_path):
+    f = open(file_path)
+    return json.load(f)
+
+def write_json(dir, file_name, json_object):
+    f = os.path.join(dir, file_name)
+    with open(f, "w") as outfile:
+        outfile.write(json_object)
```

Comment:
Right now this function is taking in a json string. It can be updated to take in a dict. You don't need to convert the dict to a string outside of this function.

```python
def write_json(dir, file_name, json_dict):
    with open(os.path.join(dir, file_name), "w") as outfile:
       json.dump(json_dict, outfile)
```

Reply:
Done